src/SFA.DAS.Authorization/Logging/AuthorizationResultLogger.cs

```diff
@@ -23,7 +23,9 @@ public AuthorizationResultLogger(IAuthorizationHandler authorizationHandler, ILo
         public async Task<AuthorizationResult> GetAuthorizationResult(IReadOnlyCollection<string> options, IAuthorizationContext authorizationContext)
         {
             var authorizationResult = await _authorizationHandler.GetAuthorizationResult(options, authorizationContext).ConfigureAwait(false);
-            var message = $"Finished running handler with prefix '{Prefix}' for options '{string.Join(", ", options)}' and context '{authorizationContext}' with result '{authorizationResult}'";
+            authorizationContext.TryGet("AccountId", out string AccountId);
+            authorizationContext.TryGet("HashedAccountId", out string HashedAccountId);
```

Comment:
once again this change is going to cause this logging to be far less useful, as some things are whitelisted by email address so omitting it removes part of the context. It is something we cannot then glean from the accountId and hashedAccountId. If we had the userRef on hand that would have been handy in place of the email address

Reply:
Apologies on approving with out knowing the context of the change. Agree with Corey. We had a conversation with Dan as well in this regards and he agreed that email on its own shouldn't be an issue.

Reply:
You could also question why log the handler running at all?  Do we need this? What is the point?